Documentation/development-flow.md

```diff
@@ -52,6 +52,46 @@ git remote -v
 ```
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+## Layout
+
+A source code layout is shown below, annotated with comments about the use of each important directory:
+
+- cmd # binaries with main entry points
+  - rook # main command entry points for operators and daemons
+    - ceph
+    - minio
+    - cockroachdb
+    - nfs
+  - rookflex
```

Comment:
Do we need to add comments for the `rookflex` listed here since it's a flex volume plugin not a trivial component? 😄

Reply:
I think it will be good to explain every component as a new contributor will be curious to know about every folder. I will update the document this weekend. For the current update on this document, I copy pasted from https://github.com/rook/rook/blob/master/design/multiple-storage-types-support.md#layout

Reply:
Thanks for the explanation! 👍